interface/forms/newpatient/common_help.php

```diff
@@ -0,0 +1,134 @@
+                
+                <p><?php echo xlt("The default is Normal, i.e. it is visible to everyone. Selecting High will restrict it to users belonging to the Physicians and Administrators groups by deafult");?>.
+                
+                <p><i class="fa fa-exclamation-triangle" style="color:red" aria-hidden="true"></i> <strong><?php echo xlt("You need administrator privileges to give access to other groups");?>.</strong>
+                
+                <p><?php echo xlt("Access can be given to other groups by editing User Memberships or Groups and Access Controls in Administration > ACL");?>.
+
+                <p><?php echo xlt("Onset/hosp. date - is used for billing hospital encounters");?>.
+
+                <p><i class="fa fa-exclamation-triangle" style="color:red" aria-hidden="true"></i> <strong><?php echo xlt("You need administrator privileges to perform the changes to the values in the drop-down boxes.");?>.</strong>
+            </div>
+            <div class= "row" id="visit_reason">
+                <h4 class="oe-help-heading"><?php echo xlt("Reason for visit");?><a href="#"><i class="fa fa-arrow-circle-up float-right oe-help-redirect" aria-hidden="true"></i></a></h4>
+                <p><?php echo xlt("A few words to indicate the reason for the visit");?>.
+
+                <p><?php echo xlt("It will be displayed in the visit summary");?>.
+
+                <p><?php echo xlt("Also provides a short description of the visit in Patient Reports");?>.
+            </div>
+            <div class= "row" id="link_issue">
+                <h4 class="oe-help-heading"><?php echo xlt("Link/Add Issues (Injuries/Medical/Allergy) to Current Visit");?><a href="#"><i class="fa fa-arrow-circle-up float-right oe-help-redirect" aria-hidden="true"></i></a></h4>
+                <p><?php echo xlt("An issue is a medical problem, allergy, medication, surgery or dental issue");?>.
+
+                <p><?php echo xlt("It is used to link problems with encounters and is also used to see which encounters are associated to a particular issue");?>.
+
+                <p><?php echo xlt("Clicking on the Add Issue button brings up the add issue pop-up box");?>.
+                <a href="#" class="css_button_small" style="float:none"><?php echo xlt("Add Issue");?></a>
+
+                <p><?php echo xlt("The default issue type is set to Problem");?>.
+
+                <p><?php echo xlt("For each issue type there is a list of commonly selected titles which are customizable for your clinic's specific needs");?>.
+
+                <p><?php echo xlt("If the desired title is not included, it can be typed into the text box below the list");?>.
+
+                <p><?php echo xlt("Select an appropriate ICD/CPT  code. This helps search for education material to give to the patient for that particular problem");?>.
```

Comment:
Remove double spacing in above constant and ensure none of the other constants in this script have double spacing

Reply:
Done